source/vocab/construct-enum-restrictions.rq

```diff
@@ -19,12 +19,12 @@ construct {
 
 (:additionalType	UNDEF	:Cartography	marc:MapsFormatType) # TODO: just make the MapsFormatType instances subclasses of Cartography?
 
-(:baseMaterial	:BaseMaterial	:Globe	marc:GlobeMediumType)
-(:baseMaterial	:BaseMaterial	:Map	marc:MapMediumType)
-(:baseMaterial	:BaseMaterial	:Microform	marc:FilmBaseType)
-(:baseMaterial	:BaseMaterial	:MovingImage	marc:FilmBaseType)
-(:baseMaterial	:BaseMaterial	:Visual	marc:ProjGraphEmulsionType)
-(:baseMaterial	:BaseMaterial	:StillImage	marc:NonProjectedType)
+(:baseMaterial	:Material	:Globe	marc:GlobeMediumType)
+(:baseMaterial	:Material	:Map	marc:MapMediumType)
+(:baseMaterial	:Material	:Microform	marc:FilmBaseType)
+(:baseMaterial	:Material	:MovingImage	marc:FilmBaseType)
+(:baseMaterial	:Material	:Visual	marc:ProjGraphEmulsionType)
+(:baseMaterial	:Material	:StillImage	marc:NonProjectedType)
```

Comment:
The drawback of changing these now is that we need to update all of those mappings in Marcframe also 🤔